src/web/core/src/main/java/org/geoserver/web/data/resource/BasicResourceConfig.java

```diff
@@ -54,7 +55,8 @@
 
     public BasicResourceConfig(String id, IModel<ResourceInfo> model) {
         super(id, model);
-
+        
+        add(new Label("storeName", model.getObject().getStore().getName()));
```

Comment:
The UI looks like this:
![image](https://user-images.githubusercontent.com/1590542/91709177-48509700-eb7a-11ea-892b-f43b99dd1f3b.png)

I would use a different styling and an information icon, something like this:
![image](https://user-images.githubusercontent.com/1590542/91709879-505d0680-eb7b-11ea-8e0a-7e4ff30864c8.png)

Reply:
Example of an info icon is available on the edit style page:
![image](https://user-images.githubusercontent.com/1590542/91709983-6ff42f00-eb7b-11ea-9b92-6a1c05d019cb.png)

Reply:
This is how the UI looks like now. 
My doubt is whether the em semantic tag is correct here...
![2020-09-14_17h07_11](https://user-images.githubusercontent.com/63857/93103566-162c5280-f6ad-11ea-8f41-e0342fe9d3d0.png)